src/supervisor/Supervisor.h

```diff
@@ -198,7 +198,19 @@ class Supervisor : public iosource::IOSource
 		 */
 		std::optional<bool> bare_mode;
 		/**
-		 * Additional script filename/paths that the node should load.
+		 * Additional script filenames/paths that the node should load
+		 * after the base scripts, and prior to any user-specified ones.
+		 */
+		std::vector<std::string> addl_base_scripts;
+		/**
+		 * Additional script filename/paths that the node should load
+		 * after any user-specified scripts.
+		 */
+		std::vector<std::string> addl_user_scripts;
+		/**
+		 * The former name for addl_user_scripts, now deprecated.  Use
+		 * addl_user_scripts instead. The scriptland equivalent has a
+		 * proper deprecation tag.
```

Comment:
Any reason to not mark the C++ side deprecated too?

Reply:
I wondered about that. It adds a bunch of warnings to our build that seemed excessive to me given the script-layer deprecation, but it would technically be correct to deprecate there as well. Happy to do as you prefer.

Reply:
It's not part of a `detail` namespace so we should probably follow the process. At worst, you can add pragmas to the Zeek code to silence the warnings for our builds.

Reply:
Added, Tim. You can see the noise here: https://cirrus-ci.com/task/6385802665000960?logs=build#L3071

Reply:
I've pushed an update with pragma use as per our discussion.